src/js/components/Drop/DropContainer.js

```diff
@@ -73,7 +73,7 @@ class DropContainer extends Component {
       container.style.maxHeight = '';
 
       // get bounds
-      const controlRect = control.getBoundingClientRect();
+      const controlRect = findVisibleParent(control).getBoundingClientRect();
```

Comment:
find visible parent might return undefined, so `getBoundingClientRect` may fail to execute. I believe this is the reason the build is failing